test/e2e/10_helm_chart.bats

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bats
+
+load lib/env
+load lib/install
+load lib/poll
+
+function setup() {
+  kubectl create namespace "$E2E_NAMESPACE"
+  install_git_srv
+  install_tiller
+  install_helm_operator_with_helm
+  kubectl create namespace "$DEMO_NAMESPACE"
+}
+
+@test "Helm chart installation smoke test" {
+  # The gitconfig secret must exist and have the right value
+  poll_until_equals "gitconfig secret" "${GITCONFIG}" "kubectl get secrets -n ${E2E_NAMESPACE} gitconfig -ojsonpath={..data.gitconfig} | base64 --decode"
+
+  poll_until_true "namespace $DEMO_NAMESPACE" "kubectl describe ns/$DEMO_NAMESPACE"
```

Comment:
Isn't this testing tiller instead of the helm operator? I would instead  check with the helm client that the release is made

Reply:
Actually, the releases are applied below, so I would simply remove this